CONTRIBUTING.md

```diff
@@ -15,43 +15,48 @@ Please note we have a code of conduct, please follow it in all your interactions
 # Contents
 
 - [Instructions](#instructions)
-  * [Notes on Creating New Buttons](#notes-on-creating-new-buttons)
-  * [Button Ideas](#button-ideas)
-  * [Create Pull Request](#create-pull-request)
-    + [Note on abandoned pull requests](#note-on-abandoned-pull-requests)
-  * [Code of Conduct](#code-of-conduct)
-    + [Our Pledge](#our-pledge)
-    + [Our Standards](#our-standards)
-    + [Our Responsibilities](#our-responsibilities)
-    + [Scope](#scope)
-    + [Enforcement](#enforcement)
+  - [Notes on Creating New Buttons](#notes-on-creating-new-buttons)
+  - [Button Ideas](#button-ideas)
+  - [Create Pull Request](#create-pull-request)
+    - [Note on abandoned pull requests](#note-on-abandoned-pull-requests)
+    - [Adding Projects that make use of sButtons](#adding-projects-that-make-use-of-sbuttons)
+    - [Adding Projects that makes use sButtons](#adding-projects-that-make-use-of-sbuttons)
```

Comment:
There are two links instead of one, please remove the second one

Reply:
On it